KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment1/Book.java

```diff
@@ -0,0 +1,47 @@
+package edu.KaylaKornelis.advancedjava.Assignment1;
+
+public class Book{
+
+    //Attributes of a Book
+    private String title;
+    private String author;
+    private String genre;
+
```

Comment:
This is OK, better would to use an interface of a more complex class. 
I'll explain this more in this week's chat.

Reply:
I had it set up with interfaces for tilte, author, and genre. But since the instructions said to create a simple class, I took those out for now. Would you like me to put that back in before merging?